.yarnrc.yml

```diff
@@ -1,2 +1,13 @@
 yarnPath: .yarn/releases/yarn-3.4.1.cjs
 nodeLinker: node-modules
+
+packageExtensions:
+  stylelint-config-recommended-scss@*:
+    peerDependencies:
+      postcss: '8'
+  stylelint-config-standard-scss@*:
+    peerDependencies:
+      postcss: '8'
+  stylelint-config-gds@*:
+    peerDependencies:
+      postcss: '8'
```

Comment:
The `.yarnrc.yml` change is one of the suggested fixes from the linked GitHub issue.

The other suggestion is to [define `peerDependencies` in package.json](https://github.com/stylelint-scss/stylelint-config-recommended-scss/pull/112/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R70-R76). We [already have `postcss` defined as a dev dependency](https://github.com/alphagov/content-tagger/blob/main/package.json#L29-L30) so it would be a case of either duplicating or moving that value.

Did you look into that option too? I'm not really sure which is best, but it certainly feels neater to me that all the dependency related definitions are all contained in the same file (`package.json`).

Reply:
Yep I did try the `peerDependencies` solution first but for some reason it didn't work!